src/shared/test/type-coercions-test.js

```diff
@@ -0,0 +1,151 @@
+import { toBoolean, toInteger, toObject, toString } from '../type-coercions';
+
+describe('sidebar/type-coercions', () => {
```

Comment:
```suggestion
describe('shared/type-coercions', () => {
```

This needs updating to reflect the new location.